### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by adjam**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+**Program Output**
+If applicable, add program output and/or backtraces
+
+**Environment (please complete the following information):**
+ - ruby-marc version
+ - ruby runtime and version (e.g. MRI 2.7, JRuby 9.2.11.0)
+ - OS
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (billdueber) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: I'm thinking maybe we don't need this :-)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Program Output**
+If applicable, add program output and/or backtraces
+
+**Environment (please complete the following information):**
+ - ruby-marc version
+ - ruby runtime and version (e.g. MRI 2.7, JRuby 9.2.11.0)
```
Comment (billdueber) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Maybe add, "The value of RUBY_DESCRIPTION is best."

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+2. Include minimal but relevant ruby-marc code that exhibits the behavor
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Program Output**
+If applicable, add program output and/or backtraces
+
+**Environment (please complete the following information):**
+ - ruby-marc version
```
Comment (billdueber) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: - ruby-marc version from MARC::VERSION

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (adjam) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: Removing, even though my phone has more cores than my laptop does.

**Pull request merged**